src/monogame/include/csharp_enums.h

```diff
@@ -0,0 +1,2104 @@
+//
+// This code is auto generated, don't modify it by hand.
+// To regenerate it run: Tools/MonoGame.Generator.CTypes
+//
+
+#pragma once
+
+#include "csharp_common.h"
+
+enum CSGraphicsBackend : csint
+{
+    DirectX = 0,
+    OpenGL = 1,
+    Vulkan = 2,
+    Metal = 3,
+};
+
+class ECSGraphicsBackend
+{
+public:
+    static const char* ToString(CSGraphicsBackend enumValue)
+    {
+        switch (enumValue)
+        {
+            case DirectX: return "DirectX";
+            case OpenGL: return "OpenGL";
+            case Vulkan: return "Vulkan";
+            case Metal: return "Metal";
+        }
+
+        return "Unknown Value";
+    }
+};
+
+enum CSMonoGamePlatform : csint
+{
+    Android = 0,
+    iOS = 1,
+    tvOS = 2,
+    DesktopGL = 3,
+    Windows = 4,
+    WindowsUniversal = 5,
+    WebGL = 6,
+    XboxOne = 7,
+    PlayStation4 = 8,
+    PlayStation5 = 9,
+    NintendoSwitch = 10,
+    Stadia = 11,
+};
+
+class ECSMonoGamePlatform
+{
+public:
+    static const char* ToString(CSMonoGamePlatform enumValue)
+    {
+        switch (enumValue)
+        {
+            case Android: return "Android";
+            case iOS: return "iOS";
+            case tvOS: return "tvOS";
+            case DesktopGL: return "DesktopGL";
+            case Windows: return "Windows";
+            case WindowsUniversal: return "WindowsUniversal";
+            case WebGL: return "WebGL";
+            case XboxOne: return "XboxOne";
+            case PlayStation4: return "PlayStation4";
+            case PlayStation5: return "PlayStation5";
+            case NintendoSwitch: return "NintendoSwitch";
+            case Stadia: return "Stadia";
+        }
+
+        return "Unknown Value";
+    }
+};
+
+enum CSContainmentType : csint
+{
+    Disjoint = 0,
+    Contains = 1,
+    Intersects = 2,
+};
+
+class ECSContainmentType

```

Comment:
Why is this in here?

Reply:
Easy way to get string value for the enum as C++ has no built in way.

Reply:
No i mean why is `ECSContainmentType` here... that isn't planned to be exposed to native code.

But i see why... your generator just dumps all enums from MG framework to this header.